packages/itwin/tree-widget/src/components/trees/stateless/categories-tree/CategoriesTree.tsx

```diff
@@ -84,14 +90,21 @@ export function StatelessCategoriesTree({
   );
 
   const getSearchFilteredPaths = useMemo<GetFilteredPathsCallback | undefined>(() => {
+    errorRef.current && setError(undefined);
```

Comment:
```suggestion
    setError(undefined);
```

No need to check if error in defined. If it's already undefined React will skip this update.

Reply:
Fixed.